### PR TITLE
Add docs on the declarative check function

### PR DESCRIPTION
**Pull request by DmitrySharabin**

### CODE REVIEW

File: docs/define/README.md [hunk 1]
```diff
@@ -131,6 +131,8 @@ All of these take parameters and return a checking function:
 There are also the following checking functions that can be used directly:
 - `equals()`: Check if the result is equal to the expected value.
 
+Instead of providing a custom checking function, you can also tweak the one used by hTest. Simply pass an object literal with desired options as the value of `check`, and hTest will produce the right checking function for you.
+
```
Comment (LeaVerou) [inline docs/define/README.md:135]: The one used by hTest -> the default one, also explain what that is!

Comment (LeaVerou) [reply docs/define/README.md:135]: What are the options??

File: docs/define/README.md [hunk 2]
```diff
@@ -177,6 +179,20 @@ export default {
...
+```js
+import Color from "../src/index.js";
+
+export default {
+	run () {
+		return new Color("srgb", [1, 0, 0]).coords;
+	},
+	expect: [1, 0, 0],
+	check: {deep: true, epsilon: .005}
```
Comment (LeaVerou) [inline docs/define/README.md:192]: Do not put check after expect, it’s more logically grouped with run

File: docs/define/README.md [hunk 3]
```diff
@@ -177,6 +179,20 @@ export default {
 }
 ```
 
+hTest can build the right checking function behind the scenes:
+
+```js
+import Color from "../src/index.js";
```
Comment (LeaVerou) [inline docs/define/README.md:185]: Color.js is a very niche example. Check out the simple examples used by other testing frameworks

**Review by LeaVerou — CHANGES_REQUESTED** [review]

**Comment by DmitrySharabin** [comment]
Thank you for your feedback. I tried to address all the issues you pointed out and am ready for the next iteration.

**Review by LeaVerou — APPROVED** [review]
LGTM. We should mention somewhere that `equals()` does deep checking.

**Pull request merged**